### PR TITLE
add note about misleading 'Prepare to launch' message

### PR DESCRIPTION
**Pull request by ncbaratta**

### CODE REVIEW

**Comment by ncbaratta** [comment]
https://deploy-preview-1315--launcher-documentation.netlify.com/docs/contrib-guide.html#verifying-pull-requests_contributing

File: docs/topics/proc_verifying-pull-requests.adoc [hunk 1]
```diff
@@ -25,6 +25,9 @@ If the button says *Unwatch*, you are already subscribed.
 NOTE: If there is no reference to an open issue and the change is not trivial, request that an issue be created.
 +
 .. Preview the rendered documentation with the changes from the pull request by navigating to the `Conversation` tab of the pull request, clicking _Show All Checks_, and _Details_ next to the revealed `deploy/netlify`.
++
+NOTE: Even though the page says "Prepare to launch..." you have arrived at the right place. Click the `Docs` link to navigate to the edited documentation.
```
Comment (inoxx03) [inline docs/topics/proc_verifying-pull-requests.adoc]: Looks good and is definitely correct, but I have a few suggestions for consistency:

> NOTE: Even though the page says _Prepare to launch..._ you are in the right place. Click _Docs_ in the top bar to navigate to the edited documentation.

**Comment by ncbaratta** [comment]
Review comments addressed.

**Comment by inoxx03** [comment]
[test]

**Pull request merged**